packages/sdk/src/protocols/light_push/light_push.spec.ts

```diff
@@ -0,0 +1,10 @@
+// TODO: add them after decoupling `BaseProtocolSDK` from LightPush
```

Comment:
unfortunately due to `LightPush` extending `BaseProtocolSDK` there is no easy way of covering it with unit tests unless `BaseProtocolSDK` is covered. 

This is one of the reasons why we shouldn't use `extend` 

Will be a follow up.

Reply:
managed to make it in this PR